src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/BaseOptions.java

```diff
@@ -295,6 +298,12 @@ public enum ModularityMismatchPolicy {
      */
     private int sourceTabSize;
 
+    /**
+     * Argument for command-line option {@code --spec-base-URI}.
```

Comment:
Have my earlier comment on the upper case in `--spec-base-URI` disappeared? I cannot seem to find it.

Reply:
Ah, found it: there are two of those cases.

Reply:
I believe these have all been fixed